content/advanced/modules/consume-example/index.md

```diff
@@ -15,6 +15,29 @@ Create a new YAML file:
 touch use-module.yaml
 :::
 
+
+   :::::tabs{variant="container"}
+	::::tab{id="cloud9" label="Cloud9"}
```

Comment:
we dont need different instructions here, just reword the 

> Open this file in your chosen text editor and paste the following CloudFormation YAML:

so it is usable for both C9 and local dev